Makefile

```diff
@@ -0,0 +1,11 @@
+check:
```

Comment:
Maybe we can add `go mod tidy/verify` to the Makefile

Reply:
Good suggestion :+1: 
What about a target called `verify` with `go mod verify`?
And a target called tidy that does `go mod tidy`, `gofmt -w .`, ` goimports -w .`

Reply:
Yep, we can add `tidy verify golang-ci` at local test

Reply:
https://github.com/layer5io/meshery-adapter-library/issues/5